test/dlc_tests/eks/pytorch/training/test_eks_pytorch_training.py

```diff
@@ -9,7 +9,6 @@
 from retrying import retry
 
 import test.test_utils.eks as eks_utils
-from dlc.github_handler import GitHubHandler
```

Comment:
So we will be not needed githandler in any of the multi node test ?

Reply:
correct. it is not required for kubeflow setup.